clients/index.js

```diff
@@ -142,7 +142,8 @@ function ApplicationClients(options) {
     });
 
     self.processReporter = ProcessReporter({
-        statsd: self.statsd
+        statsd: self.statsd,
+        handleInterval: config.get('clients.process-reporter.handle-interval', 1000)
```

Comment:
`config.get()` takes one argument. The second argument is ignored.

The semantics of `config.get()` are that if the key does not exist throw an exception.

Just remove the `1000`.